src/views/blocks/tickets/extra-available-quantity.php

```diff
@@ -11,15 +11,22 @@
  * @link {INSERT_ARTICLE_LINK_HERE}
  *
  * @since 4.9.3
- * @version TBD
+ * @since TBD Corrected amount of available/remaining tickets.
  *
+ * @version TBD
  */
 
-
+/** @var Tribe__Tickets__Ticket_Object $ticket */
 $ticket = $this->get( 'ticket' );
-if ( empty( $ticket->available() ) ) {
+
+if ( empty( $ticket->ID ) ) {
 	return;
 }
+
+/** @var Tribe__Tickets__Tickets_Handler $tickets_handler */
+$tickets_handler = tribe( 'tickets.handler' );
+
+$available = $tickets_handler->get_ticket_max_purchase( $ticket->ID );
```

Comment:
If we pass `$available` to this template, it should use that, otherwise it could then setup `$tickets_handler` and try to get tickets max purchase at that point

Reply:
updated: I passed from one template to the other